casper/src/test/scala/coop/rchain/casper/MultiParentCasperAddBlockSpec.scala

```diff
@@ -46,7 +46,10 @@ class MultiParentCasperAddBlockSpec extends FlatSpec with Matchers with Inspecto
     val testProgram = for {
       deploy <- ConstructDeploy.basicDeployData[Effect](0)
       _      <- casper.deploy(deploy)
-      block  <- casper.createBlock.map { case Created(block) => block }
+      block <- casper.createBlock.map {
+                case Created(block) => block
+                case _              => throw new RuntimeException()
```

Comment:
that's not a helpful message.

Reply:
![tests](https://user-images.githubusercontent.com/6287558/56196181-a12b9800-6036-11e9-9aa3-95af135a7c36.jpg)